resdk/resolwe.py

```diff
@@ -44,6 +44,18 @@
 TOOLS_REMOTE_HOST = os.environ.get('TOOLS_REMOTE_HOST', None)
 
 
+class ResolweResource(slumber.Resource):
```

Comment:
I can add `__all__ = ('Resolwe', )` at the beginning of the file to make it more obvious and prevent some errors.